collector/kernel/process_prober.cc

```diff
@@ -39,7 +39,7 @@ ProcessProber::ProcessProber(
   check_cb("trigger_get_pid_task()");
 
   // we can remove the probe for existing now
-  probe_handler.cleanup_probe("r_get_pid_task");
+  probe_handler.cleanup_kretprobe("get_pid_task");
```

Comment:
The root cause of the bpf_logs was that the cleanup of the get_pid_task kretprobe was failing, resulting in duplicate pid_info render messages being sent.